src/shaders/shaders.js

```diff
@@ -150,7 +151,7 @@ export default {
 };
 
 // Expand #pragmas to #ifdefs.
-function compile(fragmentSource, vertexSource, isGlobalPrelude) {
+export function compile(fragmentSource, vertexSource, isGlobalPrelude) {
```

Comment:
Is this function used somewhere outside of the module?

Reply:
This was changed to allow gl-native to compile the shadow prelude without affecting the bundle size for GL-JS based on [Karim's comment](https://github.com/mapbox/mapbox-gl-js/pull/11985#discussion_r898167645).